models/model_loader.py

```diff
@@ -0,0 +1,30 @@
+from data import data_loader
+from models import model_interface
+from models.core import logistic_regression
+from models import model_constants
+
+
+def load_model(
+    model_type: model_constants.ModelType,
+    dataset_name: data_loader.DatasetName,
+    model_name: model_constants.ModelName = model_constants.ModelName.DEFAULT,
+) -> model_interface.Model:
+    """Loads the requested model from local disk.
+
+    model_name is an identifier for the model unique within the model's family
+    and training dataset.
+
+    Args:
+        model_type: The model family type to load.
+        dataset_name: The name of the dataset the model was trained on.
+        model_name: The name of the trained models.
+
+    Returns:
+        The requested model loaded from local disk."""
+    if model_type == model_constants.ModelType.LOGISTIC_REGRESSION:
```

Comment:
This wrapping of model_constants seems like overkill where things are right now, but I imagine this decision choice made sense at the time.

Reply:
I originally used strings for the config flags for model names, model types, dataset names, and recourse types. After adding enough config strings, it was hard to remember what the exact string value for each concept was. Using enums makes it easier to keep track of.